toptal/picasso
# feat: update staging banner color and favicon

https://toptal-core.slack.com/archives/CCC3GP6CC/p1604935946061400

### Description

Use different colors for favicon to help distinguish temploys from staging.
Adjust environment banner colors according to [the design](https://app.abstract.com/projects/1b06c884-06af-482a-bf12-a82f521a19a1/branches/master/commits/ebb7cf6ef9c38f840e79addb88810916c60be421/files/a5d56a4c-1c16-4631-a2f7-8868c49147d9/layers/3B863FAB-C94C-4EE2-8061-FBFAB45778AE).

### How to test
 
- [See EnvironmentBanner on the demo page](https://picasso.toptal.net/update-staging-banner-color-and-favicon/?path=/story/components--environmentbanner)

### Screenshots

| Before.                                 | After.                                  |
| --------------------------------------- | --------------------------------------- |
| <img width="1503" alt="Screenshot 2020-11-10 at 10 06 40" src="https://user-images.githubusercontent.com/2437969/98652836-83223880-233c-11eb-826d-f1f4f3d346bb.png"> | <img width="1500" alt="Screenshot 2020-11-10 at 10 06 07" src="https://user-images.githubusercontent.com/2437969/98652792-7271c280-233c-11eb-89cd-756d7704bcad.png"> |


![Screenshot 2020-11-10 at 16 38 48](https://user-images.githubusercontent.com/2437969/98695855-681eeb00-2373-11eb-8898-64f824f0cbce.png)
![Screenshot 2020-11-10 at 16 31 36](https://user-images.githubusercontent.com/2437969/98695873-6c4b0880-2373-11eb-84c2-d11f83b4131c.png)


### Review


<details>
<summary>PR commands</summary>
<br />

List of available commands:

- `@toptal-bot run all` - Run  whole pipeline
- `@toptal-bot run danger` - Danger checks
- `@toptal-bot run lint` - Run linter
- `@toptal-bot run test` - Run jest
- `@toptal-bot run build` - Check build
- `@toptal-bot run test:visual` or `@toptal-bot run visual` - Run visual tests
- `@toptal-bot run deploy:documentation` - Deploy documentation
- `@toptal-bot run package:alpha-release` - Release alpha version

</details>
